### PR TITLE
fix succeed date_format

### PR DESCRIPTION
**Pull request by dannyroberts**
due to new submissions with microseconds

### CODE REVIEW

**Comment by czue** [comment]
(this code isn't tested)

File: custom/succeed/reports/all_patients.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from corehq.apps.reports.sqlreport import DatabaseColumn, AggregateColumn, SqlTabularReport, DataFormatter, \
     TableDataFormat
 from corehq.const import SERVER_DATETIME_FORMAT
```
Comment (czue) [inline custom/succeed/reports/all_patients.py]: possibly unused import?

Comment (dannyroberts) [inline custom/succeed/reports/all_patients.py]: Yup: https://github.com/dimagi/commcare-hq/pull/6645

**Comment by dannyroberts** [comment]
Yeah I noticed this code isn't tested But the worst that can happen that it'll continue not to work, and anyway I eyeballed it very carefully.

**Comment by czue** [comment]
sorry, i was just justifying my preemptive merge, not asking you to write a test.

**Comment by dannyroberts** [comment]
Ha, right

**Pull request merged**